analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/ClearTextProtocolsAreSensitive.cs

```diff
@@ -158,6 +159,58 @@ public void Ftp()
             "FTP://anonymous@example.com" // Noncompliant
         };
     }
+
+    [XmlRoot(ElementName = "SonarProjectConfig", Namespace = "http://www.sonarsource.com/msbuild/analyzer/2021/1")]
+    public class NamespaceLikeAssignment
+    {
+        private string NamespaceLikeField = "http://www.sonarsource.com/msbuild/analyzer/2021/1";
+        private string NamespaceLikeProperty { get; set; }
+
+        public void Foo(string NamespaceLikeArgument)
+        {
+            var NamespaceLikeVar = "http://www.sonarsource.com/msbuild/analyzer/2021/1";
```

Comment:
could you also add tests with `fooNamespace` and `FOO_NAMESPACE`?

And add some test cases:

```
using System;
using System.Xml.Serialization;

[XmlRoot(Namespace=XML_NAMESPACE)]
public class C {
    public const string XML_NAMESPACE = "http://x";
}

public static class Constants {
    public const String NAMESPACE1 = "http://x";
}
```